app/views/stages/_fields.html.erb

```diff
@@ -23,23 +23,44 @@
     </div>
   </div>
 
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-10">
-      <div class="checkbox">
-        <%= form.label :confirm do %>
-          <%= form.check_box :confirm %>
-          Confirm before deployment
-        <% end %>
+  <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+    <div class="form-group">
+      <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
+      <div class="col-lg-4">
+        <%= form.collection_select(
+          :deploy_groups, DeployGroup.all, :id, :name,
+          {
+            selected: (@stage.deploy_groups.ids if @stage && @stage.deploy_groups),
```

Comment:
Yea, there can be no @stage if it's a new Project. I got caught out on that one.